### DISCUSSION TITLE
CORS Error with SpatieMediaLibraryFileUpload and S3

### DISCUSSION THREAD

**Discussion by tomjamon** [filamentphp/filament#11202] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Hi,

I got a CORS Error with my input SpatieMediaLibraryFileUpload when loading form on FIlament.
The upload work, but the preview bug in Filament and so I'm stuck on changing the image.

`Access to fetch at 'https://....s3.fr-par.scw.cloud/367/01HNG5T...ECJ5-thumbnail.jpg' from origin 'http://localhost:8000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

**Comment by leandrocfe** [comment]
https://laracasts.com/discuss/channels/general-discussion/filepond-laravel-cors-not-sure-laravel-issue?page=1&replyId=753707

**Comment by ricventu** [comment]
Hi,
i solved setting in livewire config `temporary_file_upload.disk` to `local`

```sh
php artisan livewire:publish --config
```

```php
    'temporary_file_upload' => [
        'disk' => 'local',
        ...
```

**Reply by shankhadevpadam** [reply]
This is not worked.
